android/package.json

```diff
@@ -23,13 +23,13 @@
 		"checksum": "7798aae19b2efff53b1f18d53345291ab16a574e0524350eed3db6d45757bc476a158303b8df6c5f359439f9184818d61eba6bd820cec404e268ab0796cc7c5a"
 	},
 	"minSDKVersion": "19",
-	"compileSDKVersion": "28",
+	"compileSDKVersion": "29",
 	"vendorDependencies": {
-		"android sdk": ">=23.x <=28.x",
-		"android build tools": ">=25.x <=28.x",
-		"android platform tools": "28.x",
+		"android sdk": ">=23.x <=29.x",
```

Comment:
We don't want to change "android sdk" to "29.x" yet... because it'll make our build system target Android Q by default. I've made a separate ticket to do this in [TIMOB-26951](https://jira.appcelerator.org/browse/TIMOB-26951). Can you change it back please?
```
"android sdk": ">=23.x <=28.x",
```

Reply:
This was intentional, wouldn't we want to target by default to see what potential issues we run into? I also wanted to see what tests would fail, but Jenkins needs updating.